src/Compilers/CSharp/Test/Semantic/Semantics/LookupTests.cs

```diff
@@ -1444,6 +1444,72 @@ private void TestLookupSymbolsNestedNamespaces(List<ISymbol> actual_lookupSymbol
             Assert.Contains(aliasY, actual_lookupSymbols);
         }
 
+        [Fact]
+        public void ExtensionMethodCall()
+        {
+            var source =
+@"static class E
+{
+    internal static void F(this object o)
+    {
+    }
+}
+class C
+{
+    void M()
+    {
+        /*<bind>*/this.F/*</bind>*/();
+    }
+}";
+            var compilation = CreateCompilationWithMscorlibAndSystemCore(source);
+            var tree = compilation.SyntaxTrees[0];
+            var model = compilation.GetSemanticModel(tree);
+            compilation.VerifyDiagnostics();
+            var exprs = GetExprSyntaxList(tree);
+            var expr = GetExprSyntaxForBinding(exprs);
+            var method = (MethodSymbol)model.GetSymbolInfo(expr).Symbol;
+            var reducedFrom = method.ReducedFrom;
+            Assert.NotNull(reducedFrom);
+            Assert.Equal("E.F(object)", reducedFrom.ToDisplayString());
+        }
+
+        [WorkItem(3651)]
```

Comment:
It is not necessary for the test to run. However, this was our convention at least since we moved to the GitHub. It removes any ambiguity about what that number is referring to. It is also very convenient to copy/paste the URL, or simply double click it in VS in order to go to the issue.